pkg/util/server/error.go

```diff
@@ -34,9 +34,13 @@ type ErrorResponseBody struct {
 	Message string `json:"message"`
 }
 
+func NotFoundHandler(w http.ResponseWriter, r *http.Request) {
+	JSONError(w, 404, "page not found")
```

Comment:
`<nit>`

```suggestion
	JSONError(w, 404, "not found")
```

`</nit>`

We don't really work with "pages" - let's not confuse the operator